src/utility/killPlayerIfExhausted.js

```diff
@@ -0,0 +1,13 @@
+import { playerStatus } from '../content/playerStatus';
+import { renderScene } from './renderScene';
+
+const killPlayerIfExhausted = () => {
+  if (playerStatus.health <= 0
```

Comment:
Store critical values of stats in constant.

Reply:
done